### PR TITLE
Mvp2 343 meta tags

### PR DESCRIPTION
**Pull request by hammadgigalabs**
Task: https://wallfair-product.atlassian.net/browse/MVP2-343

**Work**

- Added express in React
- Send a request to backend to get dynamic data for events and bets
- Setting the required values in the meta tags

### CODE REVIEW

File: server/meta/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              let {
+                marketQuestion,
+                description,
+                evidenceDescription,
+                slug: betSlug,
+              } = singleBet;
+              meta[`${eventSlug}/${betSlug}`] = {
+                title: marketQuestion,
+                description: description ? description : evidenceDescription,
+                image: previewImageUrl ? previewImageUrl : meta['/'].image,
```
Comment (zametica) [inline server/meta/index.js:86]: `previewImageUrl  || meta['/'].image`

File: public/index.html [hunk 2]
```diff
@@ -1,35 +1,25 @@
...
+    <meta name="theme-color" content="#000000" />
+
+    <meta name="title" content="__title__" />
+    <meta name="description" content="__description__" />
+    <meta name="image" content="__image__" />
+    <meta name="keywords" content="__keywords__">
+
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <title>Wallfare</title>
```
Comment (zametica) [inline public/index.html:17]: Wallfair

File: server/meta/index.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  appendRoutes: async (apiPath, listPaths = []) => {
+    for (let listCounter = 0; listCounter < listPaths.length; listCounter++) {
+      // quering api to get data
+      const response = await axios.get(`${apiPath}${listPaths[listCounter]}`);
+      if (response && response.data) {
+        let dataKeys = Object.keys(response.data);
+        dataKeys.forEach(key => {
+          let singleEvent = response.data[key];
+          let { slug, name, previewImageUrl, bets, tags } = singleEvent;
+          let keywordsToUse = _.map(tags, 'name');
```
Comment (zametica) [inline server/meta/index.js:62]:
```
const eventTags = _.map(tags, 'name') || [];
const keywordsToUse = eventTags.length ? eventTags.join(', ') : meta['/'].keywords;
```
so you don't need to recheck it every time

File: server/meta/index.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+};
+
+module.exports = {
+  // Append routes
+  appendRoutes: async (apiPath, listPaths = []) => {
+    for (let listCounter = 0; listCounter < listPaths.length; listCounter++) {
+      // quering api to get data
+      const response = await axios.get(`${apiPath}${listPaths[listCounter]}`);
+      if (response && response.data) {
+        let dataKeys = Object.keys(response.data);
```
[the remark this answers is not in the document]
Comment (hammadgigalabs) [reply server/meta/index.js]: made the change, please verify

File: server/meta/index.js (hunk 1 above)
Comment (hammadgigalabs) [reply server/meta/index.js]: made the change, please verify

File: public/index.html (hunk 2 above)
Comment (hammadgigalabs) [reply public/index.html]: made the change, please verify

File: server/meta/index.js (hunk 3 above)
Comment (hammadgigalabs) [reply server/meta/index.js]: made the change, please verify

**Pull request merged**